# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-riscv.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-99.0.1/third_party/rust/nix/src/sys/ioctl/linux.rs.vanilla	2022-04-23 14:47:38.878324900 +0200
+++ firefox-99.0.1/third_party/rust/nix/src/sys/ioctl/linux.rs	2022-04-23 14:50:11.170318828 +0200
@@ -14,7 +14,7 @@
 #[doc(hidden)]
 pub const TYPEBITS: ioctl_num_type = 8;
 
-#[cfg(any(target_arch = "mips", target_arch = "mips64", target_arch = "powerpc", target_arch = "powerpc64", target_arch = "sparc64"))]
+#[cfg(any(target_arch = "mips", target_arch = "mips64", target_arch = "powerpc", target_arch = "powerpc64", target_arch = "sparc64",  target_arch = "riscv32", target_arch = "riscv64"))]
 mod consts {
     #[doc(hidden)]
     pub const NONE: u8 = 1;
